examples/playground/src/App.tsx

```diff
@@ -2,35 +2,39 @@ import type { JSONSchema7 } from 'json-schema';
 import type { UiSchema } from '@rjsf/core';
 import { VernaContextProvider } from '@openfun/verna';
 import FormWrapper from './FormWrapper';
+import TextWidget from './widgets/TextWidget';
+import PasswordWidget from './widgets/PasswordWidget';
+import './styles/verna.scss';
+import QuizWidget from './widgets/QuizWidget';
+import { useState } from 'react';
```

Comment:
Sort imports with this order

```
node_modules
aliased imports
relative imports
```

Reply:
Let me know if it's good for you ! Not used to sort import :)